static/src/javascripts/projects/common/modules/loyalty/save-for-later.js

```diff
@@ -63,7 +63,7 @@ define([
 
     SaveForLater.prototype.hasUserSavedArticle = function (articles, shortUrl) {
         return _.some(articles, function (article) {
-            return article.shortUrl === shortUrl;
+            return article.shortUrl.contains(shortUrl);
```

Comment:
There is no `String.prototype.contains`. You could use `String.prototype.indexOf`, or `_.contains`.